shared/src/packages/trucks/libs/enums/form.enum.ts

```diff
@@ -0,0 +1,21 @@
+const FormLabel = {
+  LICENSE_PLATE: 'License Plate',
+  MANUFACTURER: 'Manufacturer',
+  YEAR: 'Year',
+  CAPACITY: 'Capacity',
+  PRICE_PER_KM: 'Price per km',
+  DRIVERS: 'Drivers',
+  TOW_TYPE: 'Tow Type',
+} as const;
+
+const FormName = {
+  LICENSE_PLATE: 'licensePlateNumber',
+  MANUFACTURER: 'manufacturer',
+  YEAR: 'year',
+  CAPACITY: 'capacity',
+  PRICE_PER_KM: 'pricePerKm',
+  DRIVERS: 'drivers',
+  TOW_TYPE: 'towType',
+} as const;
+
```

Comment:
these enums are placed in shared folder. do you need them somewhere else except frontend?